packages/docs/src/routes/docs/cookbook/detect-img-tag-onload/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Cookbook | Detect img tag onLoad
+contributors:
+ - gioboa
+---
+
+import CodeSandbox from '../../../../components/code-sandbox/index.tsx';
+
+
+# Detect img tag onLoad even if it's cached
+
+When using the onLoad$ property on an \<img\> tag in a Qwik component, you might encounter an issue where the onLoad$ event doesn't fire when the image is already cached in the browser.<br/>
+In that case the onLoad$ logic is not being triggered.<br/>
```

Comment:
I don't think you need those <br/> tags here do you?

Reply:
Yup, I need it because I would like to force a new line

Reply:
Ak ok